ui/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Multi-stage build for LiteLLM UI (Dashboard + Docs)
+FROM node:18-alpine AS builder
+
+# Install dependencies and build docs
+WORKDIR /app/docs
+COPY docs/package*.json ./
+RUN npm ci --only=production
+COPY docs/ ./
+ENV DOCUSAURUS_BASE_URL=/docs/
```

Comment:
It looks like currenty this has no effect, since `docusarus.config.ts` does not read this value. You can remove this. I think it's unlikely we want to customize this